src/SWP/Bundle/CoreBundle/Migrations/2019/07/Version20190702120702.php

```diff
@@ -18,6 +18,7 @@ public function up(Schema $schema): void
         $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
 
         $this->addSql('ALTER TABLE swp_user ADD external_id VARCHAR(255) DEFAULT NULL');
+        $this->addSql('ALTER TABLE swp_route ADD COLUMN description VARCHAR(255) DEFAULT NULL');
```

Comment:
this is wrong - it should be new migration. As we have already instances with this one marked as done.